Anlab.Mvc/Models/Person.cs

```diff
@@ -8,4 +8,11 @@ public class Person
         public string Mail { get; internal set; }
         public string FullName { get; internal set; }
     }
+
+    public class ValidPerson
```

Comment:
@jSylvestre why would a ValidPerson object have IsInvalid field?  Maybe change name to DirectoryPerson?  Or maybe just DirectoryResult, since that returns the result plus an associated person.